src/Reflection/SignatureMap/SignatureMapProvider.php

```diff
@@ -47,7 +49,7 @@ private static function getSignatureMap(): array
 				throw new \PHPStan\ShouldNotHappenException('Signature map could not be loaded.');
 			}
 
-			self::$signatureMap = $signatureMap;
+			self::$signatureMap = array_change_key_case($signatureMap, CASE_LOWER);
```

Comment:
Maybe It would be better to use lowercase names directly in functionMap.php instead of converting it in runtime. What do you think?

Reply:
We'd be completely incompatible with the same signature map in Psalm and Phan. Since the changes between the projects are sporadically merged, it would create more work.

Reply:
it makes a sense